tests/test_configuration.py

```diff
@@ -658,3 +658,16 @@ def test_store_dag_code_config_when_set(self):
         self.assertTrue(conf.has_option("core", "store_dag_code"))
         self.assertTrue(store_serialized_dags)
         self.assertFalse(store_dag_code)
+
+    @conf_vars({("webserver", "site_title"): "True"})
+    def test_site_title_config_when_set(self):
```

Comment:
This year isn't needed - it is host testing the config loading which is already well tested